src/Components/calculator.js

```diff
@@ -0,0 +1,43 @@
+function Calculator() {
+  const createDigits = () => {
+    const digits = [];
+
+    for (let i = 9; i >= 1; i -= 1) {
+      digits.push(
+        <button type="button" key={i}>{i}</button>,
+      );
+    }
+
+    return digits;
+  };
+
+  return (
+    <div className="App">
+      <div className="calculator">
+        <div className="display">
+          0
+        </div>
+        <div className="ops-dig-container">
+          <div className="operators">
+            <button type="button">÷</button>
+            <button type="button">x</button>
+            <button type="button">+</button>
+            <button type="button">-</button>
+            <button type="button">=</button>
+          </div>
+
+          <div className="digits">
+            <button type="button">%</button>
+            <button type="button">+/-</button>
+            <button type="button">AC</button>
+            {createDigits()}
+            <button type="button" className="dot">.</button>
+            <button type="button" className="zero">0</button>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] [optiona] Great work implementing the changes requested, however, I would recommend you increase the width of the operation buttons to match other buttons sizes 
![image](https://github.com/Simpleshaikh1/math-magician/assets/61408860/e47417e8-16a4-4197-ae9a-7ffa8f0eb504)